magenta/pipelines/BUILD

```diff
@@ -135,8 +135,6 @@ py_library(
     srcs = ["pipelines_common.py"],
     deps = [
         ":pipeline",
-        "//magenta/music:chord_symbols_lib",
-        "//magenta/music:chords_lib",
         "//magenta/music:melodies_lib",
```

Comment:
You can also get rid of the melodies_lib dependency; not sure how I missed all of these.